README.md

```diff
@@ -88,6 +96,13 @@ In a new terminal window, run
 pnpm dev
 ```
 
+### Using your custom .mdx docs
+1. Make sure your documentation is in .mdx format. This can be done by renaming exisiting(or compatible) markdown .md file
```

Comment:
```suggestion
1. By default your documentation will need to be in `.mdx` format. This can be done by renaming existing (or compatible) markdown `.md` file.
```

Reply:
Modified